### ISSUE TITLE
[Intermittent] "Doorhanger" displays a glitch when using the shortcut keys

### ISSUE THREAD

**Issue by alexandra-martin** [mozilla/firefox-voice#239]
### Prerequisites:
Have mic permissions enabled.

### STR:
1. Press "Command-." on Mac.
2. Observe the "Doorhanger".

### Expected result:
The "Doorhanger" opens without display issues and a vocal or written command can be made.

### Actual result:
Sometimes the display of the "Doorhanger" will be darkened for a second, then the display will be back to normal.

### Notes:
Reproduced on Mac 10.14.6 with Firefox Nightly 71.0a1 (64-bit).
This is an intermittent bug. I managed to reproduce it 1/10, but sometimes it was reproduced 2/10 or even 3/10 times.

![voice 28](https://user-images.githubusercontent.com/54401091/64782570-2e373680-d56e-11e9-8a63-819480327693.gif)

**Comment by alexandra-martin** [comment]
@awallin yes, only with the shortcut, but only on Mac. Clicking the icon on both Mac and Win never gave me this issue.

**Comment by ianb** [comment]
@alexandra-martin does this only happen if you are using Dark Mode?

If so, I'm wondering if browserStyle briefly comes into play.

**Comment by alexandra-martin** [comment]
@ianb I've run into this bug also in Light Mode.

![voice 30](https://user-images.githubusercontent.com/54401091/65023229-13ccd680-d93b-11e9-86eb-7e07633c5dc7.gif)

**Comment by alexandra-martin** [comment]
I found the following update for this bug: after using the shortcut keys, the glitch can remain in the "Doorhanger". To get rid of it, the "Doorhanger" needs to be closed and opened again.
I still reproduce it 1/10,  2/10 or even 3/10 times.

![voice 42](https://user-images.githubusercontent.com/54401091/65245383-33a7fa00-daf5-11e9-8e46-e84a44c53572.gif)

**Comment by alexandra-martin** [comment]
#### Update:
Currently the glitch can be reproduced intermittently also if the "Back" button is clicked after a command is displayed in the "Doorhanger". The current issue is reproduced 1, 2 or 3 times out of 10 on Mac 10.14.6 with Firefox Nightly 72.0a1 (64-bit).
An "Internal error: Error: communicate is not defined" error was also displaying in the "Doorhanger" like in #600.

![11 21](https://user-images.githubusercontent.com/54401091/69355258-2a204700-0c8a-11ea-85d6-a134d8678770.gif)
